README.md

```diff
@@ -995,6 +995,7 @@ Table of Contents
 
 ## Analytics, Events and Statistics
 
+   * [indicative.com](https://indicative.com/) — Customer analytics platform to optimize customer engagement, increase conversion, and improve retention.
```

Comment:
the description doesn't say anything - it describes any analytics service.  Please update it to be more relevant and some details about whats free

Reply:
apologies for the inadequate description. I have added more specific information about Indicative. will happily update again if you think it is still lacking. Thanks!